geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geohash/BoundingBox.scala

```diff
@@ -72,6 +72,14 @@ case class BoundingBox(ll: Point, ur: Point) {
    */
   def contains(otherGeom: Geometry): Boolean = geom.contains(otherGeom)
 
+  /**
+   * This bounding box equals bbox if the lower left and upper right points are the same.
+   * @param bbox
+   * @return
+   */
+  def equals(bbox: BoundingBox): Boolean = (ll.getX == bbox.ll.getX) && (ll.getY == bbox.ll.getY) &&
```

Comment:
[picky]  Does this invite the possibility of rounding errors?  For example ```(0.1).toFloat = 0.1```, but ```(0.1).toFloat.toDouble = 0.10000000149011612```, which seems to suggest that initializing from a Float (that gets upcast to a Double) might not be the same as initializing from (what was meant to be) the same Double literal.  It should be possible to pick a reasonable tolerance for this comparison, yes?